package.json

```diff
@@ -67,14 +66,14 @@
     "sinon-chai": "^2.14.0"
   },
   "peerDependencies": {
-    "pg": ">=4.3.0 <8.0.0"
+    "pg": "~4.5.7 || ~5.2.1 || ~6.4.2 || >=7.1.0 <8.0.0"
```

Comment:
Seems you are trying to limit to secure versions only https://snyk.io/vuln/npm:pg
But IMO this is responsibility of developer. Not ours. And if for some reason developer can't upgrade `pg` it will also stop him from using newer versions of `node-pg-migrate`.
IMO we should limit `pg` versions based on API compatibility and leave security considerations on developer.

Reply:
Noted. Change left out of split PR.